compass/landice/tests/ismip6_forcing/atmosphere/process_smb.py

```diff
@@ -388,39 +394,84 @@ def correct_smb_anomaly_for_climatology(self,
         },
         "2300": {
             "CCSM4": {
-                "RCP85": [
-                    "AIS/Atmospheric_forcing/CCSM4_RCP85/Regridded_08km/CCSM4_8km_anomaly_1995-2100.nc",
-                    "AIS/Atmospheric_forcing/CCSM4_RCP85/Regridded_08km/CCSM4_8km_anomaly_2101-2300.nc"]
+                "RCP85": {
+                    "4km": [
```

Comment:
I wonder if this could be generalized by having a string variable `res` that could be used here like: `"AIS/Atmospheric_forcing/CCSM4_RCP85/Regridded_0" + res + "/CCSM4_ " + res + "_anomaly_1995-2100.nc"`. If that works, it would cut down on repetition, but what you have here obviously works fine.

Reply:
This is a really great suggestion, @trhille! I've looked into incorporating it, but then found that some folders have the resolution written with `0` in front (e.g., `Regridded_04km` and `Regridded_08km`), while majority of the folders had no `0` (e.g., `Regridded_4km` and `Regridded_8km`) - I'll just keep the way it is.

Reply:
I agree with @hollyhan.  We have found enough weird cases that we just don't want to try ot generalize these things.  It's not just the resolution, it would be great to automate the model name, scenario, etc. but the naming just isn't consistent enough.  It would be really important to fix this in ISMIP7!